app/scripts/modules/core/cluster/allClustersGroupings.component.ts

```diff
@@ -0,0 +1,14 @@
+import { module } from 'angular';
+
+export const ALL_CLUSTERS_GROUPINGS_COMPONENT = 'spinnaker.core.cluster.allClustersGroupings.component';
+
+module(ALL_CLUSTERS_GROUPINGS_COMPONENT, [])
+.component('allClustersGroupings', {
+  templateUrl: require('./allClustersGroupings.component.html'),
```

Comment:
How do we feel about inlining smallish templates into the component source file?